.github/workflows/publish.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
```suggestion
on:
  release:
    types: [created]
```

Reply:
i think we're unlikely to create tags that are not releases, but i would say the release event is more specific and accurate

Reply:
I prefer being intentional as well.  This way someone can't accidently push a tag that generates a release.